src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -8179,6 +8179,7 @@ namespace SurfaceGeometry {
                 // N1: sky view factor
                 if (!state.dataIPShortCut->lNumericFieldBlanks(1)) {
                     state.dataSurface->SurroundingSurfsProperty(Loop).SkyViewFactor = state.dataIPShortCut->rNumericArgs(1);
+                    state.dataSurface->SurroundingSurfsProperty(Loop).IsSkyViewFactorSet = true;
```

Comment:
The `IsSkyViewFactorSet` variable is set to `false` if the sky view factor field is blank.